packages/rdma-core/rdma-core.spec

```diff
@@ -0,0 +1,165 @@
+Name: %{_cross_os}rdma-core
+Version: 53.1
+Release: 1%{?dist}
+Summary: RDMA core userspace libraries and daemons
+License: MIT OR GPL-2.0-only
+Source0: https://github.com/linux-rdma/rdma-core/releases/download/v%{version}/rdma-core-%{version}.tar.gz
+Source100: rdma-core-tmpfiles.conf
+
+BuildRequires: cmake >= 2.8.11
+BuildRequires: %{_cross_os}libnl-devel
+BuildRequires: %{_cross_os}glibc-devel
+Requires: %{_cross_os}libnl
+
+%description
+%{summary}.
+
+%package devel
+Summary: RDMA core development libraries and headers
+# For now we are only providing the libibverbs-devel as it
+# will be needed to build EFA Device Plugin.
+Provides: %{_cross_os}libibverbs-devel
+
+%description devel
+%{summary}.
+
+%prep
+%autosetup -n rdma-core-%{version} -p1
+
+%build
+%{cross_cmake} . \
```

Comment:
There was a recent change in the Bottlerocket SDK to provide a "toolchain file" for CMAKE: 

https://github.com/bottlerocket-os/bottlerocket-sdk/blob/c22140d86d95179f1270848c82e2df29e6a6314b/macros/shared#L208

You could use that instead and save yourself the extra CMAKE flags. You still need to provide `ENABLE_STATIC` and the `NO_*` flags.